vignettes/summarizeOverlaps.Rmd

```diff
@@ -0,0 +1,328 @@
+---
+title: "Counting reads with `summarizeOverlaps`"
+author:
+- name: Valerie Obenchain
+- name: Rohit Satyam
+  affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+clean: false
+date: "`r format(Sys.Date(), '%d %B, %Y')`"

```

Comment:
This vignette has both an "edited date" and a "compiled date". Please modify the date as

    date: "Edited: April 2016; Compiled: `r format(Sys.Date(), '%d %B, %Y')`"